example/decentralized-renderer/templates/govtechDemoCert/common/SimplePrivacyFilterBanner.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FunctionComponent } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEdit } from "@fortawesome/free-solid-svg-icons";
+import { css } from "@emotion/core";
+
+interface SimplePrivacyFilterBannerProps {
+  /**
+   * handler called when toggle edition is requested
+   */
+  onToggleEditable: () => void;
+  className?: string;
+}
+
+const style = css`
+  @media print {
+    display: none;
+  }
+  background-color: whitesmoke;
+  display: flex;
+  justify-content: space-between;
+  padding: 20px;
+  .icon {
+    cursor: pointer;
+  }
+`;
+/**
+ * Banner with icon to toggle certificate obfuscation mode
+ */
+export const SimplePrivacyFilterBanner: FunctionComponent<SimplePrivacyFilterBannerProps> = ({
```

Comment:
Wondering if this name is still appropriate for this component and file. Maybe something like `WarningBanner`?

Reply:
this component is only for the example provided, it's the SimplePrivacyFilterBanner which is used in our demo certificate and was reused there. I didnt notice when I brought it here

Reply:
Yea i got that, but do we want to give it a better name? People will be less confused that way. Can imagine people trying to figure how does it do privacy filtering lol

Reply:
Ok will rename to banner

Reply:
Done